packages/constellation/src/components/Primitives/Divider/Divider.tsx

```diff
@@ -0,0 +1,38 @@
+import { Root } from '@radix-ui/react-separator'
+import cx from 'classnames'
+import React, { forwardRef } from 'react'
+
+import { DividerOrientations, DividerVariants, Props } from './Divider.types'
+
+/**
+ * The Divider component visually or semantically separates content.
+ *
+ * `ref`s are forwarded to the Radix `root` element.
+ */
+
+const variantClasses: Record<DividerVariants, string> = {
+  distinct: 'bg-background-light',
+  medium: 'bg-medium',
+  subtle: 'bg-grey-300',
+}
+const orientationClasses: Record<DividerOrientations, string> = {
+  horizontal: 'h-[1px]',
+  vertical: 'h-full w-[1px]',
```

Comment:
Tailwind has built in, but slightly hidden, classes for 1px sizing:

```suggestion
  horizontal: 'h-px',
  vertical: 'h-full w-px',
```

Reply:
Sounds good 👍